wagl/modtran.py

```diff
@@ -50,24 +67,22 @@ def prepare_modtran(acquisitions, coordinate, albedos, basedir, modtran_exe):
         if not exists(modtran_work):
             os.makedirs(modtran_work)
 
-        out_fname = pjoin(modtran_work, 'mod5root.in')
-        with open(out_fname, 'w') as src:
-            src.write(POINT_ALBEDO_FMT.format(p=coordinate,
-                                              a=albedo.value) + '\n')
-
         symlink_dir = pjoin(modtran_work, 'DATA')
+
         if exists(symlink_dir):
             os.unlink(symlink_dir)
 
         os.symlink(data_dir, symlink_dir)
 
-        out_fname = pjoin(modtran_work, acq.spectral_filter_file)
+        out_fname = pjoin(modtran_work, acq.spectral_filter_file)  # filter file name needs to be put in DATA directory
+        print(out_fname)
```

Comment:
I'd remove the print. If it is needed for debug purposes, then using the logging library will be more preferential.